docs/operating-scylla/procedures/backup-restore/backup.rst

```diff
@@ -73,12 +73,12 @@ Incremental Backup
 
 **Procedure**
 
-| 1. In the ``/etc/scylla/scylla.yaml`` file set the ``incremental backups`` parameters to ``true`` and restart the Scylla service. Snapshot are created under Scylla data directory ``/var/lib/scylla/data``
+| 1. In the ``/etc/scylla/scylla.yaml`` file set the ``incremental backups`` parameters to ``true`` and restart the Scylla service. Incremental backups are created under Scylla data directory ``/var/lib/scylla/data``
 | with the following structure:
-| ``keyspace_name/table_name-UUID/backups/backups_name``
+| ``keyspace_name/table_name-UUID/backups``
 
 | For example:
-| ``/mykeyspace/team_roster-91cd2060f99d11e6a47a000000000000/backups/1437827672721``
+| ``/mykeyspace/team_roster-91cd2060f99d11e6a47a000000000000/backups``
```

Comment:
I think the example you changed meant to show one of the files inside the backups/ subdirectory, not the name of the subdirectory.

Reply:
I beg to differ.
`1437827672721` looks like a timestamp like the one automatically assigned to snapshots.

Reply:
files would be hardlinks the sstable components, flat in a single directory.